client/landing/gutenboarding/components/plans/plans-table/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import PlanItem from './plan-item';
+import { useSelect } from '@wordpress/data';
+import { STORE_KEY as PLANS_STORE } from '../../../stores/plans';
+
+export interface Props {
+	selectedPlanSlug: string;
+	onPlanSelect: ( planSlug: string ) => void;
+}
+
+const Plans: React.FunctionComponent< Props > = ( { selectedPlanSlug, onPlanSelect } ) => {
+	const supportedPlans = useSelect( ( select ) => select( PLANS_STORE ).getSupportedPlans() );
+
+	return (
+		<div className="plans-table">
+			<div className="plans-table__body">
+				<div className="plans-table__items">
+					{ supportedPlans.map( ( plan, i ) => (
+						<PlanItem
+							key={ i }
```

Comment:
Can we use plan slug as `key`? Always go with index as last resort. https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
something like this would work:

```suggestion
							key={ plan.getStoreSlug() }
```